Jenkinsfile_CNP

```diff
@@ -0,0 +1,18 @@
+#!groovy
+
+properties([
+  // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
+  pipelineTriggers([cron('H 05 * * *')])
+])
```

Comment:
Do we need a cron job to trigger the pipeline?

Reply:
commented out the offending line for now